schemas/v2/index.json

```diff
@@ -147,7 +147,7 @@
                             "type": "array",
                             "items": {
                                 "type": "string",
-                                "pattern": "^[\\w.-]+\\/[\\w.-]+$"
+                                "pattern": "^[\\w.-]+(\\/[\\w.-]+)*$"
```

Comment:
Since this is a glob pattern now, I don't think it's worth having a `pattern` here anymore, so we can delete this